### ISSUE TITLE
Update List Item documentations lacks information for complex columns

### ISSUE THREAD

**Issue by Mentate** [microsoftgraph/microsoft-graph-docs#5747]
Issue: Update List item doesn't give an example on how to update a complex column in a list. For example, I am trying to add users to a "Person or Group" column, but I don't know how the JSON should be formatted. I've tried to match the formatting that was used to do this with the SharePoint REST API, but that only returns errors. Any guidance in the documentation for columns other than primitive types would be very helpful. 

Article: https://github.com/microsoftgraph/microsoft-graph-docs/blob/master/api-reference/v1.0/api/listitem-update.md

**Comment by joefedorowicz** [comment]
I don't think this can be done as currently constituted but those in charge of documentation should highlight which columns cannot be modified (or created). Includes Person/Group, Managed Metadata, Multiple Choice Column, Lookup...for instance.
